### DISCUSSION TITLE
How to configure custom endpoints with graphiql in server side?

### DISCUSSION THREAD

**Discussion by GayathriRaj** [graphql-java/graphql-java#2287] [Q&A]
Hi, I have to configure below endpoints to expose graphql services and I am using graphiql library to execute the queries. And, I have done below configuration in the application.properties file to customize the default endpoint but still it points to /graphiql. Can someone please help me on this? Thanks.

graphql.servlet.mapping: /api/service
graphql.servlet.enabled= true
graphiql.mapping: /graphiql
graphiql.endpoint.graphql: /api/service

**Comment by ooga** [comment]
Hi @GayathriRaj,
graphql-java is just a library and doesn't configure any endpoint.

From your config, I will bet you are using https://github.com/graphql-java-kickstart/graphql-spring-boot
It will be better to ask your question on this repo
